setup.py

```diff
@@ -81,7 +81,7 @@
 setuptools_args = {}
 install_requires = setuptools_args['install_requires'] = [
     'ipython>=4.0.0',
-    'traitlets>=4.1.0',
+    'traitlets>=4.2.0',
```

Comment:
What in this PR changes the minimum version to 4.2? Aren't all these APIs in 4.1?